src/components/noteText/NoteText.scss

```
@@ -4,4 +4,20 @@
   min-width: 390px;
   max-height: auto;
   height: auto;
+  width: auto;
+  min-width: 500px;
+  max-width: 600px;
+  &__show {
```

It's better to have a blank line between stylings of current block and its selectors

```suggestion
  max-width: 600px;
  
  &__show {
```